### PR TITLE
User notifications with local storage docs

### PR DESCRIPTION
**Pull request by dvoytenko**
Will defer the merge for a bit, but please review when you can.

### CODE REVIEW

File: extensions/amp-user-notification/amp-user-notification.md [hunk 1]
```diff
@@ -116,6 +119,9 @@ will be passed in future requests to data-show-if-href)
     ```
   - `POST response` should be a 200 HTTP code and no data is expected back.
 
+If not specified, AMP will only not send a request upon dismissal and will only store "dismissed"
```
Comment (erwinmombay) [inline extensions/amp-user-notification/amp-user-notification.md]: is this right, "will only not" ?

Comment (dvoytenko) [inline extensions/amp-user-notification/amp-user-notification.md]: Fixed.

File: extensions/amp-user-notification/amp-user-notification.md [hunk 2]
```diff
@@ -131,6 +137,23 @@ will be passed in future requests to data-show-if-href)
...
+A notification is shown when:
+
+1. There's no record locally that the user has dismissed the notification with the
+specified ID.
+2. When specified, `data-show-if-href` endpoint returns `{ "showNotification": true }`.
+
+When notification is dismissed:
+
+1. AMP stores the "dismiss" record locally for the specified ID. This will prevent the
+notification from being shown again.
```
Comment (erwinmombay) [inline extensions/amp-user-notification/amp-user-notification.md]: is there a storage limit?

Comment (dvoytenko) [inline extensions/amp-user-notification/amp-user-notification.md]: Hmm. Always, but none that would affect this docs, I don't think.

**Comment by erwinmombay** [comment]
@dvoytenko LGTM

**Comment by dvoytenko** [comment]
@Meggin PTAL. This is coming very soon.

**Comment by Meggin** [comment]
Thanks for heads up. Can I update validation content to reflect this? If it's really soon, should be safe to do so, as we can hold off on pushing updates to doc server a day.

**Comment by dvoytenko** [comment]
My estimate is that it will launch this Thursday or Friday at the latest.

**Comment by Meggin** [comment]
Perfect. Seems smart in my next rounds of edits to reflect these changes
then.

**Pull request merged**